docs/installing-open-mpi/configure-cli-options/misc.rst

```diff
@@ -34,6 +34,18 @@ above categories that can be used with ``configure``:
   .. danger:: The heterogeneous functionality is currently broken |mdash|
               do not use.
 
+* ``--enable-progress-threads``
+* ``--disable-progress-threads``:
+  Enable or disable (default = ``enabled``) software-based progress thread
+  for each MPI process to execute the internal communication progression
+  engine. When enabled at build time, the thread can be activated (default =
+  deactivated) at runtime by setting environment variable
+  ``OMPI_ASYNC_PROGRESS`` or ``OPAL_ASYNC_PROGRESS`` to a non-zero value.
```

Comment:
Can we make this a proper MCA parameter rather than naked env variables?  MCA parameters are The Open MPI Way -- they can be set in a variety of ways (including via env variables).

Reply:
I've pushed a new version with MCA variables `opal_async_progress` and its synonym `mpi_async_progress`, add in `opal/runtime/opal_params_core.c`